test/util/mvc/controllers/plainHTMLController.js

```diff
@@ -7,4 +7,18 @@ module.exports = (router) => {
     // send the plain HTML page back
     res.sendFile(htmlPath)
   })
+
+  router.route('/HTMLTest/nested').get((req, res) => {
+    // save the path of the plain HTML page
+    const htmlPath = path.join(__dirname, '../views/plainHTMLTest.html')
```

Comment:
To make these more re-useable later and semantically correct we might want new .html documents later.

Let me know and I can update